### DISCUSSION TITLE
VIM mode cursor

### DISCUSSION THREAD

**Discussion by semi710** [starship/starship#3471] [Q&A]
Is it possible to change cursor on the basis of the mode i am in.
like block for normal mode and line cursor for insert mode?

**Comment by chipbuster** [comment]
Unfortunately, this isn't handled by starship. You'll have to look at the settings for whatever shell you're using.

**Comment by semi710** [comment]
I am using zsh shell in kde konsole.
[Typewritten](https://github.com/reobin/typewritten) this is a prompt that i used to use and this had the functionality to change cursor so i thought that starship may have too.
